torch/csrc/jit/mobile/register_mobile_ops.cpp

```diff
@@ -7,31 +7,219 @@ using torch::jit::peek;
+}
+
 at::Tensor optional_to_tensor(c10::optional<at::Tensor> v) {
   return v.has_value() ? *v : at::Tensor();
 }
+}
 
 static auto registry0 = torch::RegisterOperators().op(
   "_aten::add.Tensor",
   torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
-  [](at::Tensor a, at::Tensor b, at::Scalar c) ->at::Tensor {
+  [](at::Tensor a, at::Tensor b, at::Scalar c) -> at::Tensor {
     return at::add(a, b, c);
   })
 ).op(
   "_aten::add.Scalar",
   torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
-  [](at::Tensor a, at::Scalar b, at::Scalar c) ->at::Tensor {
+  [](at::Tensor a, at::Scalar b, at::Scalar c) -> at::Tensor {
     return at::add(a, b, c);
   })
 ).op(
-  "_aten::_convolution",
+  "_aten::add_.Tensor",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+                                           [](at::Tensor a, at::Tensor b, at::Scalar c) -> at::Tensor {
+                                             return at::add(a, b, c);
+                                           })
+).op(
+  "_aten::adaptive_avg_pool2d(Tensor self, int[2] output_size) -> Tensor",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+  [](c10::OperatorKernel* kernel, Stack* stack) {
+  #ifdef USE_STATIC_DISPATCH
+   at::AutoNonVariableTypeMode non_var_type_mode(true);
+  #endif
+   auto result_ = at::adaptive_avg_pool2d(
+       (std::move(peek(*stack, 0, 2))).toTensor(),
+       (std::move(peek(*stack, 1, 2))).toIntListRef()
+       );
+   drop(*stack, 2);
+   pack(*stack, std::move(result_));
+  })
+).op(
+  "_aten::mm",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+  [](at::Tensor a, at::Tensor b) -> at::Tensor {
+    return at::mm(a, b);
+  })
+).op(
+  "_aten::_convolution(Tensor input, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, bool transposed, int[] output_padding, int groups, bool benchmark, bool deterministic, bool cudnn_enabled) -> Tensor",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+  [](c10::OperatorKernel* kernel, Stack* stack) {
+#ifdef USE_STATIC_DISPATCH
+     at::AutoNonVariableTypeMode non_var_type_mode(true);
+#endif
+     auto result_ = at::_convolution(
+         (std::move(peek(*stack, 0, 12))).toTensor(),
+         (std::move(peek(*stack, 1, 12))).toTensor(),
+         toOptionalTensor((std::move(peek(*stack, 2, 12)))),
+         (std::move(peek(*stack, 3, 12))).toIntListRef(),
+         (std::move(peek(*stack, 4, 12))).toIntListRef(),
+         (std::move(peek(*stack, 5, 12))).toIntListRef(),
+         (std::move(peek(*stack, 6, 12))).toBool(),
+         (std::move(peek(*stack, 7, 12))).toIntListRef(),
+         (std::move(peek(*stack, 8, 12))).toInt(),
+         (std::move(peek(*stack, 9, 12))).toBool(),
+         (std::move(peek(*stack, 10, 12))).toBool(),
+         (std::move(peek(*stack, 11, 12))).toBool()
+         );
```

Comment:
chatted offline - I think it's a bit redundant to manually write these lines, but it's fine if this is just temporary as it's going to be replaced by aten->c10 migration work in future.